src/main/java/core/basesyntax/utils/transaction/handlers/impl/SupplyOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.utils.transaction.handlers.impl;
+
+import core.basesyntax.storage.impl.StorageImpl;
```

Comment:
According to the checklist, the Storage should not be tied to its implementation. It's better to use the interface `Storage` instead of the concrete class `StorageImpl` to follow the Dependency Inversion Principle of SOLID.